src/Services/VoucherValidation/WinstUitJeWoning.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+
+namespace Tychovbh\Mvc\Services\VoucherValidation;
+
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Illuminate\Support\Arr;
+
+class WinstUitJeWoning implements VoucherValidationInterface
+{
+    /**
+     * @var Client
+     */
+    private $client;
+
+    /**
+     * WinstUitJeWoning constructor.
+     * @param Client $client
+     */
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Check if voucher is valid
+     * @param string $voucher
+     * @param array $data
+     * @return array
+     * @throws GuzzleException
+     */
+    public function validate(string $voucher, array $data): array
+    {
+        $data['verify'] = 1;
+        $data['label'] = $voucher;
+
+        return $this->request('get', $data);
+    }
+
+    /**
+     * Uses the voucher
+     * @param string $voucher
+     * @param array $data
+     * @return array
+     * @throws GuzzleException
+     */
+    public function use(string $voucher, array $data = []): array
+    {
+        $data['webshop'] = 1;
+        $data['label'] = $voucher;
+
+        return $this->request('post', $data);
+    }
+
+    /**
+     * Does request to service
+     * @param string $method
+     * @param array $params
+     * @return array
+     * @throws GuzzleException
+     */
+    private function request(string $method, array $params): array
+    {
+        $config = config('mvc-voucher-validation.providers.WinstUitJeWoning');
+        $options = [];
+        $multipart = [];
+
+        if (!empty($params)) {
+            if ($method === 'post') {
+                $index = 0;
+                foreach ($params as $key => $value) {
+                    $data = [
+                        'name' => $key,
+                        'contents' => $key === 'receiptFile' ? fopen($value, 'r') : $value
+                    ];
+                    Arr::set($multipart, $index, $data);
```

Comment:
mooi dat je gebruik maakt van Arr::set maar in dit geval is het eenvoudiger om daar niet gebruik van te maken, want dan hoef je de $index niet bij te houden:
$multipart[] = $data;
dit voegt data toe aan het einde van de array.
Ook dit mag op 1 regel:
$multipart[] = [
   'name' => '',
  ...
];